linux/ibus-kmfl/src/kmflutil.c

```diff
@@ -146,11 +146,41 @@ gchar * kmfl_get_icon_file(KInputMethod * im)
     return full_path_to_icon_file;
 }
 
+gchar * kmfl_get_ldml_file(KInputMethod * im)
+{
+    gchar * full_path_to_ldml_file=NULL, *p, *filename;
+    struct stat filestat;
+
+    p=rindex(im->keyboard_visualkeyboard,'.');
```

Comment:
This should be below the initial `if` statement. We should also really do a sanity check to ensure that what follows the `.` is `kvk` so that we don't end up, for example, with a path to a folder with a dot in it.

Reply:
Good point. Changes made. I assume that should still be reviewed even though the PR status is still 'Approved' even after adding a commit?

Reply:
I approved it because I figured you'd be able to make the change without needing a re-review. But LGTM :)